packages/cli/src/index.ts

```diff
@@ -134,22 +134,24 @@ async function execCmd(args: string[]) {
     throw new ExitError(1);
   }
   let shouldFix = things[0] === "fix";
-  let { packages, root, tool } = (await getPackages(
+  let { tool, packages, rootPackage, rootDir } = (await getPackages(
     process.cwd()
   )) as PackagesWithConfig;
 
   let options: Options = {
     ...defaultOptions,
-    ...root.packageJson.manypkg,
+    ...(rootPackage ? rootPackage.packageJson.manypkg : {}),
   };
 
   let packagesByName = new Map<string, Package>(
     packages.map((x) => [x.packageJson.name, x])
   );
-  packagesByName.set(root.packageJson.name, root);
+  if (rootPackage) {
+    packagesByName.set(rootPackage.packageJson.name, rootPackage);
+  }
   let { hasErrored, requiresInstall } = runChecks(
     packagesByName,
-    root,
+    rootPackage!,
```

Comment:
This non-null assertion doesn't look right. I think I managed to juggle things around a little bit to fix it here:
https://github.com/elliot-nelson/manypkg/pull/3